indico/modules/rb_new/util.py

```diff
@@ -104,3 +105,9 @@ def get_linked_object(type_, id_):
                         SessionBlock.session.has(db.and_(~Session.is_deleted,
                                                          Session.event.has(is_deleted=False))))
                 .first())
+
+
+def is_booking_start_within_grace_period(start_dt, allow_admin=False):
+    if allow_admin:
```

Comment:
:warning: you aren't checking if the user actually IS an admin

Reply:
Ahh, right!